tests/factories/organization.js

```diff
@@ -0,0 +1,26 @@
+import FactoryGuy from 'ember-data-factory-guy';
+
+FactoryGuy.define('provider', {
```

Comment:
This should be "organization" then you'll be able to make("organization") in the tests

Reply:
No, can't use organisation as there is no organisation model.

Reply:
You're right, my mistake I misunderstod what this was doing.
Maybe you can use polymorphic model.
https://github.com/danielspaniel/ember-data-factory-guy#polymorphic-model